### ISSUE TITLE
Put caret after curlies when completing "interface{}"

### ISSUE THREAD

**Issue by deadok22** [go-lang-plugin-org/go-lang-idea-plugin#2797]
On type of "{" after 'interface', the caret is put between the curly braces. 

It's really annoying, especially when initializing slices of ```interface{}```:
```go
func Test() {
  ifaces := []interface<caret>
}
``` 

The caret is put inside the curly braces and I instinctively start typing in values, which breaks parsing. And it's not immediately apparent what's wrong, so I end up scratching my head for several seconds =)

**Comment by sjamesr** [comment]
What if you want to type `[]interface{foo() void}{ ... }`?

**Comment by dlsniper** [comment]
Based on this:
> it's not immediately apparent what's wrong

Maybe it would be better to detect these cases and offer a quick fix rather than change the behavior?

**Comment by deadok22** [comment]
@sjamesr wow, I didn't know golang has this syntax! Thanks! =)

**Comment by deadok22** [comment]
@dlsniper yes, that would help. However, that would require the plugin's parser to parse the invalid code.
